internal/internal.go

```diff
@@ -40,7 +41,7 @@ type evalResult struct {
 	revision   string
 	decisionID string
 	txnID      uint64
-	decision   bool
+	decision   interface{}
```

Comment:
can this just be a `string`? rather than an `interface{}`

Reply:
Tried to go there, actually, but faced issues trying to assign string to interface on https://github.com/Yelp/opa-istio-plugin/blob/master/internal/internal.go#L260

And then trying to address that I faced `cannot use &result.decision (type *string) as type *interface {} in assignment` 

and then trying to cast to interface{} to address that I faced `invalid type assertion: result.decision.(<inter>) (non-interface type string on left)`

So I gave up and went with what you see.

Reply:
Got this working with your help, something something type coercion